cmd/bacalhau/docker_run.go

```diff
@@ -73,7 +73,8 @@ type DockerRunOptions struct {
 	NodeSelector     string   // Selector (label query) to filter nodes on which this job can be executed
 
 	Image      string   // Image to execute
-	Entrypoint []string // Entrypoint to the docker image
+	Entrypoint string   // Entrypoint to the docker image
+	Cmd        []string //Command arguments for Entrypoint
```

Comment:
The field in the JobSpecDocker is called `Parameters` – should this be the same?